Manifest.toml

```diff
@@ -0,0 +1,92 @@
+[[Base64]]
+uuid = "2a0f44e3-6c83-55bd-87e4-b1978d98bd5f"
+
+[[Compat]]
+deps = ["Base64", "Dates", "DelimitedFiles", "Distributed", "InteractiveUtils", "LibGit2", "Libdl", "LinearAlgebra", "Markdown", "Mmap", "Pkg", "Printf", "REPL", "Random", "Serialization", "SharedArrays", "Sockets", "SparseArrays", "Statistics", "Test", "UUIDs", "Unicode"]
+git-tree-sha1 = "2d9e14d19bad3f9ad5cc5e4cffabc3cfa59de825"
+uuid = "34da2185-b29b-5c13-b0c7-acf172513d20"
+version = "1.3.0"
```

Comment:
What is the procedure for maintaining this file? Do we have to manually `Pkg.update()` when Compat gets updated?

Reply:
This gets updated automatically when you run `Pkg` operations locally in this project.

Reply:
So yes, this requires manual updates. In what workflow do you need a manifest file for the package? `Pkg.test` doesn't require it.

Reply:
I use a `Manifest.toml` so that my travis builds against JuMP master. I couldn't figure out how to do this in `Project.toml`:
https://github.com/odow/Kokako.jl/blob/d6e48925b450de46f3fa17adf494c10deca59471/Manifest.toml#L86-L92